miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/UserInfoBridge.kt

```diff
@@ -145,6 +164,7 @@ internal class UserInfoBridge {
         const val ERR_GET_ACCESS_TOKEN = "Cannot get access token:"
         const val ERR_ACCESS_TOKEN_NO_PERMISSION =
             "Permission has not been accepted yet for getting access token."
+        const val ERR_ACCESS_TOKEN_NOT_MATCH_MANIFEST = "Not match the audience and scope in manifest."
```

Comment:
Maybe we can write: "The audience and scopes have not been matched in the manifest."